packages/reference-apps/checksum-sequence/index.ts

```diff
@@ -1,23 +1,36 @@
 /* eslint-disable no-loop-func */
 import { TransformApp } from "@scramjet/types";
-import { StringDecoder } from "string_decoder";
-var crypto = require("crypto");
-const decoder = new StringDecoder("utf-8");
-
-export = async function(_stream: any) {
+// import { StringDecoder } from "string_decoder";
+import * as crypto from "crypto";
+import { readFileSync } from "fs";
+// const decoder = new StringDecoder("utf-8");
 
+export = async function(_stream: any, ffrom = `${__dirname}/data.json`) {
     this.logger.info("Sequence Checksum is called.");
-    let out = "";
 
-    for await (const chunk of _stream) {
-        this.logger.info("Sequence Checksum received chunk: " + chunk.toString());
-        out += decoder.write(chunk);
-    }
-    out += decoder.end();
-    const hex = crypto.createHash("md5").update(out).digest("hex");
+    const jsonToString = readFileSync(ffrom).toString();
```

Comment:
In the Instance we should read from the _stream (input).
The idea was to check whether we can send all data without any packet lost to the Instance using input stream (/input endpoint). Let's talk tomorrow morning and look into it together :)